jkamz/Questioner
# #163446512 add question record

### What does this PR do?
This PR creates the endpoint for adding a question

### Description of the tasks to be completed
create an endpoint for posting questions to given meetups
### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/Questioner`

`$ cd Questioner`

Check out to the PR's branch
`$ git checkout ft-add-question-record-163446512`

### What are the relevant Pivotal Tracker stories?
#### 163446512
#### https://www.pivotaltracker.com/story/show/163446512

### Screeenshots

![meetupdbqu](https://user-images.githubusercontent.com/11965186/51677768-ba024f80-1feb-11e9-9fd9-0a3176c90622.png)
